mappings/net/minecraft/client/color/block/BlockColors.mapping

```diff
@@ -1,2 +1,14 @@
 CLASS net/minecraft/class_1327 net/minecraft/client/color/block/BlockColors
 	FIELD field_27100 providers Lnet/minecraft/class_6202;
+	METHOD method_28942 getColot (Lnet/minecraft/class_4541;Lnet/minecraft/class_5086;Lnet/minecraft/class_6183;)I
+		ARG 1 state
+		ARG 2 world
+		ARG 3 pos
+	METHOD method_28943 getProvidedColor (Lnet/minecraft/class_4541;Lnet/minecraft/class_5090;Lnet/minecraft/class_6183;I)I
```

Comment:
`getColor`

Reply:
this only checks the color provider
the other getColor checks both color provider and material

Reply:
But upstream :) and either way both of them give u the color